src/scss/components/optional/_previous-order-card.scss

```diff
@@ -0,0 +1,108 @@
+/**
+* Components > Previous Order Card
+* =================================
+*
+* Example: Homepage – "Order it Again" block.
+*
+* The `c-previousOrderCard` component is an optional mixin within Fozzie.
```

Comment:
Is it worth naming this as just an `orderCard`?  Just wondering if naming it like this means we couldn't use it elsewhere if we wanted to reuse the styles again?